cluster/mesos/docker/certs/rebuild.sh

```diff
@@ -0,0 +1,27 @@
+#!/bin/bash
+
+# Copyright 2014 The Kubernetes Authors All rights reserved.
```

Comment:
Looks like there's 900+ examples in the repo using 2014... Is it supposed to be 2015 for just new files?